custodia/httpd/server.py

```diff
@@ -319,21 +319,17 @@ def handle_one_request(self):
                 self.send_error(500)
                 self.wfile.flush()
                 return
+
+            output = response.pop('output', None)
             self.send_response(response.get('code', 200))
             for header, value in six.iteritems(response.get('headers', {})):
                 self.send_header(header, value)
             self.end_headers()
-            output = response.get('output', None)
```

Comment:
At one point during the development of the patch, it made sense. It is no longer necessary but also harmless.